src/content/nft/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Non-fungible tokens (NFT)
+description: An overview of NFTs on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/eth2/core.png
+summaryPoints:
+  [
+    "A way to turn anything unique into an Ethereum-based asset.",
+    "NFTs are giving more power to content creators than ever before.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+NFTs are currently taking the digital art and collectibles world by storm. Digital artists are seeing their lives change thanks to huge sales to a new crypto-audience. And celebrities are joining in as they spot a new opportunity to connect with fans.
+
+Here's what you need to know about NFTs.
+
+## What's an NFT?
```

Comment:
Worth [defining/explaining "fungibility"](https://www.investopedia.com/terms/f/fungibility.asp) given that it's part of the name? Or was the thinking to intentionally avoid it, given it's a relatively esoteric term to begin with?